pkg/ipmi/dev.go

```diff
@@ -13,6 +13,69 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func (cc CompletionCode) Error() string {
+	switch cc {
+	case IPMI_CC_OK:
```

Comment:
this would be good as a map

map[CompletionCode]string

Reply:
Done.